budgetApp1/budgetApp1/AccountViewController.swift

```diff
@@ -9,35 +9,70 @@ import UIKit
 
 class AccountViewController: UIViewController {
 
+    // MARK: - variables
     @IBOutlet private var tableView: UITableView!
-    
     private lazy var viewModel = AccountViewModel(repository: AccountRepository(), delegate: self)
     
+    // MARK: - UISpecs Dependency
+    private let uiSpecs = UISpecs()
+    
+    // MARK: - functions
+    private func setupTableView() {
+        tableView.delegate = self
+        tableView.dataSource = self
+        tableView.register(AccountViewCell.nib(), forCellReuseIdentifier: NibIdentifiers.accountViewCellIdentifier)
+        tableView.separatorStyle = .singleLine
+        tableView.separatorColor = UIColor.black
+        tableView.separatorInset = UIEdgeInsets(top: 0, left: 16, bottom: 0, right: 16)
+    }
+    
     override func viewDidLoad() {
         super.viewDidLoad()
         setupTableView()
-        viewModel.getAccounts()
+        viewModel.fetchAccounts()
     }
     
-    private func setupTableView() {
-        tableView.delegate = self
-        tableView.dataSource = self
+    override func viewWillAppear(_ animated: Bool) {
+        super.viewWillAppear(animated)
+        if let selectedIndexPath = tableView.indexPathForSelectedRow {
+            tableView.deselectRow(at: selectedIndexPath, animated: animated)
+        }
     }
 }
 
 // MARK: - TableView Delegate
-
 extension AccountViewController: UITableViewDelegate, UITableViewDataSource {
+    
     func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
-        return viewModel.AccountListCount
+        viewModel.accountListCount
+    }
+    
+    func tableView(_ tableView: UITableView, heightForRowAt indexPath: IndexPath) -> CGFloat {
+        100.0
     }
     
     func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
-        guard let cell = tableView.dequeueReusableCell(withIdentifier: "defaultCell") as? AccountCell else { return UITableViewCell() }
-        guard let account = viewModel.account(atIndex: indexPath.item) else { return UITableViewCell() }
+        guard let cell = tableView.dequeueReusableCell(withIdentifier: NibIdentifiers.accountViewCellIdentifier) as? AccountViewCell else { return UITableViewCell() }
+        guard let account = viewModel.account(atIndex: indexPath.row) else { return UITableViewCell() }
```

Comment:
These two guards can be a joined guard instead of having two separate ones

```suggestion
        guard let cell = tableView.dequeueReusableCell(withIdentifier: NibIdentifiers.accountViewCellIdentifier) as? AccountViewCell,
        let account = viewModel.account(atIndex: indexPath.row) else { return UITableViewCell() }
```